andela/blackninjas-backend
# #171240760 Admin should be able to assign users to their managers

#### What does this PR do?
It enables the admin to assign users their managers
#### Description of Task to be completed?
- admin can assign user to existing user(manager)
- write unit tests
#### How should this be manually tested?
- Clone this repo
- make sure you have all packages installed by typing `npm install`
- add all tables and seeders by typing `npm run db-migrate && npm run db-seed-dev`
- navigate to this route `/api/v1/users/user/:userId(the id of the user who will get a manager)`
- add the token of the admin in the headers and in the body add the manager ID and send and you should get a 200 HTTP status code with data created otherwise you will get the relevant error message.
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#171240760](https://www.pivotaltracker.com/story/show/171240760)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/51836855/74422499-64e3e680-4e57-11ea-926c-6f86a0408c36.png)

![image](https://user-images.githubusercontent.com/51836855/74434716-a2eb0580-4e6b-11ea-8218-3b802a53e5e4.png)

![image](https://user-images.githubusercontent.com/51836855/74434738-ad0d0400-4e6b-11ea-980d-62c23c23fb5b.png)

#### Questions:
